Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/Microsoft.Toolkit.Forms.UI.Controls.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.Controls</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Forms.UI.Controls</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit Windows Forms Controls</Title>
+    <Description>This library provides XAML user controls for Windows Forms. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>WebView Win32 WindowsForms</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>$(DefineConstants);DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.Toolkit.Win32.UI.Controls\**\*.cs" LinkBase="Shared" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Design" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xaml" />
+    <Reference Include="WindowsBase" />
+    <PackageReference Include="System.Net.Http" Version="4.0.0" />
+    <PackageReference Include="System.Runtime.WindowsRuntime" PrivateAssets="All">
+      <Version>4.0.0</Version>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Toolkit.Forms.UI.XamlHost\Microsoft.Toolkit.Forms.UI.XamlHost.csproj" />
```

Comment:
This needs to have a target min version defined--The XamlHost control needs a version later than 17134 (e.g. if we don't install the RS5 SDK this won't build)